### PR TITLE
Improve UI by adding 2nd paginator and making 1st one movable

### PR DESCRIPTION
**Pull request by limonte**
This improvement is useful for all users as we're using vertical space more efficiently. Especially it is important for mobile devices.

### CODE REVIEW

**Comment by limonte** [comment]
Come on, this is the only one old and unmerged PR, let's finish this! :)

I'll show you the improvement in gifs:

**Before:**

recorded

**After:**

recorded

A bit handier, isn't it? :)

File: src/components/ItemList.vue [hunk 1]
```diff
@@ -16,11 +10,13 @@
         </transition-group>
       </div>
     </transition>
+    <newsListNav :page="page" :maxPage="maxPage" :type="type" v-if="displayedPage > 0"></newsListNav>
```
Comment (yyx990803) [inline src/components/ItemList.vue]: Can we use `kebab-case` here to keep it consistent with the rest of the project?

**Comment by yyx990803** [comment]
Hey sorry for letting this sit for so long - the gif helps a lot, looks great. Just a minor code style change needed.

File: src/components/ItemList.vue (hunk 1 above)
Comment (limonte) [inline src/components/ItemList.vue]: Done.

**Comment by yyx990803** [comment]
Thanks!

**Pull request merged**